populus/cli/main.py

```diff
@@ -26,50 +18,51 @@
 
 @click.group(context_settings=CONTEXT_SETTINGS)
 @click.option(
-    '--config',
-    '-c',
-    'config_file_path',
+    '--project-root-dir',
+    '-p',
+    'project_root_dir',
+    help=(
+        "Specify a populus project root directory"
+    ),
+    type=click.Path(exists=True),
+)
+@click.option(
+    '--user-config',
+    '-g',
+    'user_config_path',
+    help=(
+        "Specify a global populus configuration file, instead of the default"
+    ),
+    type=click.Path(exists=True),
+)
+@click.option(
+    '--warnings',
+    'show_warnings',
     help=(
-        "Specify a populus configuration file to be used.  No other "
-        "configuration files will be loaded"
+        "Show all warnings"
     ),
-    type=click.Path(exists=True, dir_okay=False),
+    default=False,
```

Comment:
I actually think this should be defaulted to `True` if I understand what this is doing (causing `warning.warn` messages to be printed to stdout/err?)